includes/wikia/nirvana/WikiaResponse.class.php

```diff
@@ -454,6 +458,9 @@ public function sendHeaders() {
 				$this->setContentType( 'text/javascript; charset=utf-8' );
 			} else if ( $this->getFormat() == WikiaResponse::FORMAT_HTML ) {
 				$this->setContentType( 'text/html; charset=utf-8' );
+			} else if ( $this->getFormat() == WikiaResponse::FORMAT_INVALID ) {
```

Comment:
Done. Also converted the format to lowercase in case something passes an uppercase value and switching to `===` would break it.